src/Microsoft.AspNetCore.Server.Testing/Deployers/RemoteWindowsDeployer/StartServer.ps1

```diff
@@ -28,19 +31,28 @@ IF (-Not [string]::IsNullOrWhitespace($environmentVariables))
 	}
 }
 
+# Temporary workaround for issue https://github.com/dotnet/cli/issues/2967
+if ($executablePath -ne "dotnet.exe"){
```

Comment:
This fix is for standalone only...for portable apps, the shared runtime needs to be fixed one time